3-networks/modules/vpn-ha/variables.tf

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "default_region1" {
+  type        = string
+  description = "Default region 1 for Cloud Routers"
+}
+
+variable "default_region2" {
+  type        = string
+  description = "Default region 2 for Cloud Routers"
+}
+
+variable "network_name" {
+  type        = string
+  description = "Network Name"
+}
+
+variable "vpc_name" {
```

Comment:
Why is there vpc_name *and* network_name?

Reply:
Only `vpc_name` will gonna be input.